### PR TITLE
Add logging to USA facts indicator

### PR DESCRIPTION
**Pull request by benjaminysmith**
### Description
Add logging to USA facts indicator

### Changelog
- Add logging for elapsed time, CSVs processed, max lag, etc.
- Update prod param for logging output

### Fixes 
- Incremental on #689

### CODE REVIEW

**Review by sgsmob — APPROVED** [review]
Just one question with an obvious fix or no fix depending on the answer but otherwise LGTM.

File: usafacts/delphi_usafacts/run.py [hunk 1]
```diff
@@ -99,14 +108,24 @@ def run_module():
...
-        create_export_csv(
+        exported_csv_dates = create_export_csv(
             df,
             export_dir=export_dir,
             start_date=SMOOTHERS_MAP[smoother][3](export_start_date),
             metric=metric,
             geo_res=geo_res,
             sensor=sensor_name,
         )
+        if not exported_csv_dates.empty:
```
[the remark this answers is not in the document]
Comment (krivard) [reply usafacts/delphi_usafacts/run.py:119]: (`if not X empty` should probably translate to `if X`)

[the remark this answers is not in the document]
Comment (benjaminysmith) [reply usafacts/delphi_usafacts/run.py:119]: Unfortunately not -- I tried this at first and you get:

`ValueError: The truth value of a Series is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().`

**Comment by krivard** [comment]
Is this ready to merge?

**Comment by benjaminysmith** [comment]
Yes please.

**Pull request merged**